frontend/src/pages/home/aiFlows/DeployAndMonitorGallery.tsx

```diff
@@ -32,26 +57,27 @@ const DeployAndMonitorGallery: React.FC<{ onClose: () => void }> = ({ onClose })
         isOpen
       />,
     );
-  }
 
-  infoItems.push(
-    <InfoGalleryItem
-      key="model-deploy"
-      data-testid="ai-flows-model-deploy-info"
-      title="Deploying models"
-      resourceType={ProjectObjectType.deployingModels}
-      sectionType={SectionType.serving}
-      description={
-        <Content>
-          <Content component="small">
-            Deploy models to test them and integrate them into applications. Deploying a model makes
-            it accessible via an API, enabling you to return predictions based on data inputs.
+    infoItems.push(
```

Comment:
This "deploying models" info item should appear if model serving is enabled even if modelmesh is not enabled. (i.e. if only kserve is enabled). That used to be true simply because this component was only rendered if model serving was enabled and this was outside the `if (modelMeshEnabled)` that you moved it into.

I think you need to add `const kServeEnabled = servingPlatformStatuses.kServe.enabled;` above next to line 13 where `modelMeshEnabled` is defined, and then this "deploying models" info item should be pushed `if (modelMeshEnabled || kServeEnabled)`.